evaluate/metrics/assessment.go

```diff
@@ -0,0 +1,40 @@
+package metrics
+
+// AssessmentKey is a description for a numerical assessment value.
+type AssessmentKey string
+
+// allAssessmentKeys holds all assessment keys.
+var allAssessmentKeys []AssessmentKey
+
+func registerAssessmentKey(key string) AssessmentKey {
+	assessment := AssessmentKey(key)
+	allAssessmentKeys = append(allAssessmentKeys, assessment)
+
+	return assessment
+}
+
+var (
+	// AssessmentKeyNoExplanations means that a model did not produce additional explanations.
+	AssessmentKeyNoExplanations = registerAssessmentKey("no-explanations")
+)
+
+// Assessments holds numerical assessment metrics.
+type Assessments map[AssessmentKey]uint
+
+// Add sums two assessment objects.
```

Comment:
-> `merges two assessments into a new assessment and returns it.` ?

Reply:
Will not return it as discussed above.